<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ScreenwritingClass</class>
 <widget class="QMainWindow" name="ScreenwritingClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1307</width>
    <height>917</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Screenwriting</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTableView" name="tableView">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>11</y>
      <width>811</width>
      <height>631</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="description_line_edit">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>660</y>
      <width>361</width>
      <height>61</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="act_line_edit">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>740</y>
      <width>361</width>
      <height>61</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>670</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Description</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>760</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Act</string>
    </property>
   </widget>
   <widget class="QPushButton" name="propose_button">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>710</y>
      <width>121</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Propose</string>
    </property>
   </widget>
   <widget class="QPushButton" name="accept_button">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>710</y>
      <width>121</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Accept</string>
    </property>
   </widget>
   <widget class="QPushButton" name="develop_button">
    <property name="geometry">
     <rect>
      <x>1060</x>
      <y>350</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Develop</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>990</x>
      <y>160</y>
      <width>261</width>
      <height>131</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="save_plot_button">
    <property name="geometry">
     <rect>
      <x>1030</x>
      <y>710</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Save Plot</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1307</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="Screenwriting.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>propose_button</sender>
   <signal>clicked()</signal>
   <receiver>ScreenwritingClass</receiver>
   <slot>propose()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>543</x>
     <y>774</y>
    </hint>
    <hint type="destinationlabel">
     <x>721</x>
     <y>774</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tableView</sender>
   <signal>clicked(QModelIndex)</signal>
   <receiver>ScreenwritingClass</receiver>
   <slot>idea_selected()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>682</x>
     <y>263</y>
    </hint>
    <hint type="destinationlabel">
     <x>1122</x>
     <y>733</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>accept_button</sender>
   <signal>clicked()</signal>
   <receiver>ScreenwritingClass</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>804</x>
     <y>780</y>
    </hint>
    <hint type="destinationlabel">
     <x>883</x>
     <y>819</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>develop_button</sender>
   <signal>clicked()</signal>
   <receiver>ScreenwritingClass</receiver>
   <slot>open_edit_window()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1142</x>
     <y>406</y>
    </hint>
    <hint type="destinationlabel">
     <x>1124</x>
     <y>530</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>save_plot_button</sender>
   <signal>clicked()</signal>
   <receiver>ScreenwritingClass</receiver>
   <slot>save_plot()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1083</x>
     <y>763</y>
    </hint>
    <hint type="destinationlabel">
     <x>993</x>
     <y>648</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>propose()</slot>
  <slot>idea_selected()</slot>
  <slot>accept()</slot>
  <slot>open_edit_window()</slot>
  <slot>save_plot()</slot>
 </slots>
</ui>
